src/wishlist.tm

```diff
@@ -21,7 +21,55 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
+
+  <hrule>
+
+  An user-friendly interface to create/modify shortcuts.
+
+  <hrule>
+
+  Being able to easily create macros, at least non-parametrical ones, with a
+  single menu action. It is already possible, but the syntax not easy for
+  begginers (and non-programmers in general).
+
+  <hrule>
+
+  Being able to adjust the math typesetting parameters. For example, by
+  default, one may find formulas too compact, and would like to have more
+  space around (in)equality signs.
+
+  <hrule>
+
+  A variant for underbraces and overbraces with <verbatim|\\smash> ;
```

Comment:
I do not understand what you mean here. Can you explain what is the purpose and the use case of this feature?

Reply:
![image](https://user-images.githubusercontent.com/34941891/98749375-6d038f00-23b3-11eb-8379-995da7d298de.png)

Reply:
ok, good. why you do not add this example to your wish? it makes clear what you mean. Also we could have an article about "typographical defects in texmacs"

Reply:
Did it already